test/octokit/get-octokit-throttle-options.test.ts

```diff
@@ -0,0 +1,21 @@
+import pino from "pino";
+import { getOctokitThrottleOptions } from "../../src/octokit/get-octokit-throttle-options";
+
+describe(".getOctokitThrottleOptions()", () => {
+  test("No redis configured in options or env", () => {
+    expect(getOctokitThrottleOptions({ log: pino() })).toBeUndefined();
+  });
+
+  test("redis configured in options", () => {
+    const redisConfig = {
+      host: "test",
+    };
+    const throttleOptions = getOctokitThrottleOptions({
+      log: pino(),
+      redisConfig,
+    });
+    throttleOptions?.connection.disconnect();
+    expect(throttleOptions?.onAbuseLimit).toBeDefined();
+    expect(throttleOptions?.onRateLimit).toBeDefined();
+  });
+});
```

Comment:
I'd like to test it with an integration test that is using only the public APIs, but I wasn't able to do that in a way that it reproduces the error.

The throttle callbacks are set here:
https://github.com/probot/probot/blob/310dd3eeea7a4dca9c54fe09e8f9d4a5a0d5a0ac/src/octokit/probot-octokit.ts#L21-L42

It must be getting lost in your case, but I'm not sure in what constellation yet